app/helpers/static_pages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```

Comment:
delete this file because not use

Reply:
![Screenshot from 2021-08-25 14-01-45](https://user-images.githubusercontent.com/87750131/130734957-9d496d24-2ac1-4cbf-9f90-c0679438c644.png)